### PR TITLE
add entries for TKET and pytket

### PR DESCRIPTION
**Pull request by CalMacCQ**
Adding entries for pytket and the TKET compiler.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [Perceval](https://github.com/Quandela/Perceval) - [Quandela](https://www.quandela.com)'s software library for programming realistic photonic quantum computers.
 - [ProjectQ](https://github.com/ProjectQ-Framework/ProjectQ) - Hardware-agnostic framework with compiler and simulator with emulation capabilities.
 - [PyQudit](https://github.com/Ordoptimus/pyqudit) - Python package for generalized and universal versions of quantum gates in N-dimensions.
+- [pytket](https://docs.quantinuum.com/tket/) - Quantum computing toolkit for building, compiling and executing quantum circuits (developed by Quantinuum).
```
Comment (dlyongemallo) [inline README.md:63]: I'm surprised that this wasn't on the list already. (I'm also surprised that entries involving Quantinuum aren't using the Oxford comma.)

File: README.md [hunk 2]
```diff
@@ -255,6 +256,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [Catalyst](https://github.com/PennyLaneAI/catalyst) - AOT/JIT compiler for hybrid quantum computing beyond NISQ. Written in MLIR. Python frontend via [PennyLane](https://pennylane.ai).
 - [QCOR](https://github.com/ORNL-QCI/qcor) - C++ language extension and associated compiler implementation for hybrid quantum-classical programming.
 - [ScaffCC](https://github.com/epiqc/ScaffCC) - Compilation, analysis and optimization framework for the Scaffold quantum programming language.
+- [TKET](https://github.com/CQCL/tket) - C++ library for placement, routing and optimization of quantum circuits (developed by Quantinuum).
```
Comment (dlyongemallo) [inline README.md:259]: Should [TKET v2](https://github.com/CQCL/tket2) be added under "Rust" as well?

File: README.md (hunk 1 above)
Comment (CalMacCQ) [reply README.md]: Haha fixed thanks. I work in Cambridge ;)

File: README.md (hunk 2 above)
Comment (CalMacCQ) [reply README.md]: Good point, added

**Comment by dlyongemallo** [comment]
Thanks for the contribution @CalMacCQ!

**Comment by CalMacCQ** [comment]
> Thanks for the contribution @CalMacCQ!

No problem. Is there a separate place that should be updated for the entires to appear here on this page?
https://qosf.org/project_list/

**Comment by dlyongemallo** [comment]
There's code in [the qosf.org repo](https://github.com/qosf/qosf.org) which is supposed to run to update those entries, but it looks like it hasn't been run for two years. I guess you can either raise an issue there, or run the [script](https://github.com/qosf/qosf.org/blob/master/qosf.org/scripts/github_scraping.py) yourself and send a PR.

**Pull request merged**